### ISSUE TITLE
Doesn't drag when dragging on links

### ISSUE THREAD

**Issue by nhantrn** [davidparsson/scrollbar-anywhere#51]
I notice it doesn't drag when I grab on links. Is this on purpose? I use Firefox's GrabnDrag and it doesn't have this behavior. Please add option to allow dragging when grabbing on links.

**Comment by davidparsson** [comment]
@nhantrn, yes, this is by design. If not, this plugin would interfere with link clicking. I'm not sure but there might be a way to get around this by being clever.

However, I'm not currently spending much time at all on this extension, so pull requests are welcome.

**Comment by Droyk** [comment]
can't you add a user choice, an option to make this possible it's really frustrating that you can't drag over the images.

**Comment by davidparsson** [comment]
@Droyk, the reason for this is actually technical. I'm not sure how to easily fix this so that it works consistently, and sadly I don't have the time to make a technical investigation.

If anybody has any input, or the time to make a pull request, that would be much appreciated.

**Comment by Droyk** [comment]
No problem David, just remember whenever you can extract some little time from your busy life give this extension a shot because there are still many users using this extension & love every part of it and are also willing to wait for your updates so what I simply want to say is Don't abandon it, please.

**Comment by davidparsson** [comment]
Sure, I won't! But for a while I'll probably have to keep the updates to what's critically necessary.

Thanks for your appreciation!
